CanHazFunny/CanHazFunny/JokeServiceInterface.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace CanHazFunny;
+
+public  interface IJokeServiceInterface
```

Comment:
```suggestion
public  interface IJokeService
```
Because it's prefixed with an `I` we know it's an interface and saying interface is redundant